extensions/vscode/src/eventErrors.ts

```diff
@@ -49,5 +49,5 @@ export const handleEventCodedError = (
   }
 
   const unknownErrMsg = emsg.data.error || emsg.data.message;
-  return `Unknown error: ${unknownErrMsg}`;
+  return `${unknownErrMsg}`;
```

Comment:
Without the introductory text, now this can be `return emsg.data.error || emsg.data.message;`

Reply:
Great observation. I'll make the change.